tests/behat/features/ra_bug-pieter.feature

```diff
@@ -0,0 +1,33 @@
+Feature: A RA manages tokens tokens registered in the selfservice portal
+  In order to manage tokens
+  As a RA
+  I must be able to manage second factor tokens in RA
```

Comment:
Please rewrite the feature description to match the issue that is verified in this feature. This seems to be a copy paste from ra_vet.feature